recepty/levne_sushi_po_cesku.txt

```diff
@@ -0,0 +1,68 @@
+Ingredience:
+
+300 g rýže
+3 ks párek
+5 ks okurka
+20 g škrob
+
+
+Postup:
+
+Nejprve uvaříme rýži a vaříme ji tak dvakrát déle než obvykle, 
+aby se pěkně rozvařila a byla měkoučká. Mezitím si můžeme oloupat 
+okurku tak, aby byla slupka dlouhá a ponecháme si ji na zabalení.
+
+Po uvaření rýže do ní přidáme škrob, aby se více lepila. Následně
+položíme na slupku rýži a na rýži párek. Zabalíme slupku tak, aby
+vytvořila krásná sushi kolečka s párkem uprostřed. Uděláme toto 
+několikrát, dokud nebudeme mít hodně koleček. Nezapomeňte, že 
+pokud nám kolečka nebudou držet, můžeme je sepnout párátkem, 
+ale jde to i bez něj.
+
+Levné sushi po česku podáváme na stůl k jídlu. 
+Sushi kolečka můžeme namáčet v křenu a také v kečupu.
+
+Inspirováno:
+https://recepty.vareni.cz/levne-sushi-po-cesku/
+
+Obrázek:
+*(((((((((((((/////,  *******,,,,,,,,,,,,,,,,,,,,,,,,,,,,,,,,,,,,,,,,,,,,,,,,,,,,,,,,,,,,,,,,,,,,,,
+(((((((((((((///*  ,******,,,,,,,,,,,,,,,,,,,,,,,,,,,,,,,,,,,,,,,,,,,,,,,,,,,,,,.,,,,,,,,,,,,,,,,,,
+((((((((((////,  ******,,,,,,,,,,,,,,,,,,,,,,,,,,,,,,,,,,/@&/,*(%/,,,,,,,,,,,,,,,,,,,,,,,,,,,,,,,,,
+(((((((((///*  *******,,,,,,,,,,,,,,,,.,/,,,,,,,,,,,,,,,,,..**,*/,,//&,,,,,,,,,,,,,,,,,,,,,,,,,,,,,
+((((((((//*  *******,,,,,,,,,,/...    .,...,*,,,,,,,**,,,,,..**/.,,*,,/(%,,,,,,,,,,,,,,,,,,,,,,,,,,
+((((((///  .******,,,,,,,,,*,.*,..,.     ...,/((,,,#*,,,,,,.,,,.,,,,*//*(%@,,,,,,,,,,,,,,,,,,,,,,,,
+(((((//*  *//****,,,,,,,*(*. . .. .......*.,,,.,%,,#,,.,*,,,,,,,,.,,,,,,,*%#@,,,*,,,,,,,,,,,,,,,,,,
+((((//, ,//****,,,,,,,**.....,,,....,,.,.,,*,.,,(((*/*,,***,***/(*/,,/***,/*#@*******,*,,,,,,,,,,,,
+((///. ,//****,,,,,,,**,,*,*//**/*///*..,,,,..,*/(%/.,,,*.,/(////(///((//,((##(***********,,,,,,,,,
+(((/. *///***,,,,,,,*#*...*/////////////*,,,,*,/((&**,,,*////////((*////*/***((@***********,,,,****
+/(/, *//*****,,,,,,,(//..*/*/////*/*///#(,,,,,,*,(&(*,,,**((///(((//(//(////*(*&(*************,****
+//* *//*****,,,,,,,&/. ,**////////*/*//*/(/*..,*/#@(,,***//////((/////(///*,,//(/******************
+//..///****,,,,,,,,%,,,.,***/*//////**/**,,,.,,,,#@(*/(.**(/(/(((///#//*//*(,,*(%******************
+/*.///*****,,,,,,,,(%(,,.,**/////*******..,,,,.**(#.(,,***(///(////////*/*,*,*/#(******************
+/.*//*****,,,,,,,,,,@#/,,*/**/*********.,,,**,,*/#&*,*****/((/(///////////*/*,#%*****************,*
+*.///****,,,,,,,,,,,,@##,,***********#,,,,,,,**//@#(/******,*//*****/*(***,/((&//*****/**********,*
+,*///****,,,,,,,,,,,,#%,.,,.,..,*/,**,,,,,,.,,,,/%@#///******/*,,**,,,**(**/#/*/***************,,
+,///****,,,,,,,,,,,,,,&*,,.*,*/*,,. ..,,.*/*,,*/&@@@@/,,,/**,***,,*,****//((///*/*************,.,
+,///*****,,,,,,,,,,,,,,,&@&*,,*....,...***,*((/%@@@@@@#/(/**//((*****//(/%@((((////*************,,,
+,///*****,,,,,,,,,,,,,,,*/&&*//*,*/.**/,/##///(/*//#((/@/(,*%,,*&@@@@#((((//////************,.,
+*///****,,,,,,,,,,,,,,,,,*//#&@//@@/@/%@@@//@/@*/((*&/(@/((*/(,,(((((%((/(((((/////***********, .
```

Comment:
jak si to udelal/a ?

Reply:
https://manytools.org/hacker-tools/convert-images-to-ascii-art/ něbo tak něco

Reply:
diky 🗡  👍  :D